From richard@rsk.demon.co.uk  Sun Feb  7 20:21:47 2010
From: Richard Kennedy <richard@rsk.demon.co.uk>
Date: Thu, 04 Feb 2010 11:33:49 +0000
Subject: Staging: wlan-ng: minor cleanups
To: Greg Kroah-Hartman <greg@kroah.com>
Message-ID: <1265283229.2175.10.camel@localhost>


Remove iw_handle_def private initialisations as they are no longer need
these.

Signed-off-by: Richard Kennedy <richard@rsk.demon.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/p80211wext.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/staging/wlan-ng/p80211wext.c
+++ b/drivers/staging/wlan-ng/p80211wext.c
@@ -1744,11 +1744,7 @@ static iw_handler p80211wext_handlers[]
 
 struct iw_handler_def p80211wext_handler_def = {
 	.num_standard = ARRAY_SIZE(p80211wext_handlers),
-	.num_private = 0,
-	.num_private_args = 0,
 	.standard = p80211wext_handlers,
-	.private = NULL,
-	.private_args = NULL,
 	.get_wireless_stats = p80211wext_get_wireless_stats
 };
 
